data/natural/preprocess_arxiv.py

```diff
@@ -161,17 +161,17 @@ def latex_to_markdown_with_pandoc(latex_text):
                 flags=re.MULTILINE | re.DOTALL | re.IGNORECASE,
             )
     except Exception as e:
-        print(f"An error occurred during Pandoc conversion: {e}")
+        #print(f"An error occurred during Pandoc conversion: {e}")
         return None
 
     return markdown_text
 
 
 def clean_ds():
-    ds_arxiv = load_dataset("EleutherAI/proof-pile-2", "arxiv")
+    ds_arxiv = load_dataset("togethercomputer/RedPajama-Data-1T", "arxiv")
     clean_entries = []
 
-    for entry in ds_arxiv["train"]:
+    for entry in tqdm(ds_arxiv["train"], total=len(ds_arxiv["train"])):
```

Comment:
Should we also get the val/test splits?

Reply:
I can double check but on Redpyama there were none

Reply:
EAI has some: https://huggingface.co/datasets/EleutherAI/proof-pile-2/tree/main/arxiv

Reply:
But not much more information in their paper.

Reply:
ok, I can see how they constructed those. I ended up directly using the redpajama one because EAI one made some issues